lambdas/management/status.go

```diff
@@ -0,0 +1,75 @@
+package management
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/weka/aws-tf/modules/deploy_weka/lambdas/common"
+	"github.com/weka/go-cloud-lib/connectors"
+	"github.com/weka/go-cloud-lib/lib/jrpc"
+	"github.com/weka/go-cloud-lib/lib/weka"
+	"github.com/weka/go-cloud-lib/logging"
+	"github.com/weka/go-cloud-lib/protocol"
+)
+
+type StatusRequest struct {
+	UsernameId        string   `json:"username_id"`
+	PasswordId        string   `json:"password_id"`
+	AdminPasswordId   string   `json:"admin_password_id"`
+	BackendPrivateIps []string `json:"backend_private_ips"`
```

Comment:
no need for the admin password
lets be explicit and call it `DeploymentUsernameId` and `DeploymentPasswordId` and use the same for the enironment variables names.

Reply:
taking back the admin password part, we will need it for upgrade of old deployments